Formula/erlang_ls.rb

```diff
@@ -0,0 +1,44 @@
+class ErlangLs < Formula
+  VERSION = "0.21.2".freeze
+  desc "Erlang Language Server"
+  homepage "https://erlang-ls.github.io/"
+  url "https://github.com/erlang-ls/erlang_ls/archive/refs/tags/#{VERSION}.tar.gz"
+  sha256 "8a0466f525091a483401a958c6c1f85cd3203c6103b0fbabf784be449031b42c"
+  license "Apache-2.0"
+
+  depends_on "erlang@23" => :build
+
+  resource "rebar3" do
+    url "https://github.com/erlang/rebar3/releases/download/3.18.0/rebar3"
+    sha256 "f3e2641be923ce23076ce4843ee61c63fb392bc6c44dc9d129e4b31f7e136ff0"
+  end
+
+  def patch_app_versions
+    resource_files = [
+      "apps/els_lsp/src/els_lsp.app.src",
+      "apps/els_dap/src/els_dap.app.src",
+    ]
+
+    resource_files.each do |file_name|
+      inreplace(file_name) { |f| f.gsub!("{vsn, git}", "{vsn, \"#{VERSION}\"}") }
+    end
+  end
+
+  def install
+    resource("rebar3").stage buildpath
+
+    chmod "+x", buildpath/"rebar3"
+
+    ENV.append_path "PATH", buildpath
+
+    patch_app_versions
+
+    system "make"
+    bin.install "_build/default/bin/erlang_ls"
+    bin.install "_build/dap/bin/els_dap"
```

Comment:
```suggestion
    system "make", "PREFIX=#{prefix}", "install"
```
Does this work?

Reply:
I tried this before, it won't work:
```
Installing escript...
cp: /usr/local/bin/erlang_ls: Operation not permitted
make: *** [install] Error 1
/usr/local/Homebrew/Library/Homebrew/shims/shared/git --version
/usr/local/Homebrew/Library/Homebrew/shims/shared/curl --version
/usr/local/Homebrew/Library/Homebrew/ignorable.rb:29:in `block in raise'
BuildError: Failed executing: make PREFIX=/usr/local/Cellar/erlang_ls/0.21.2 install
```

Reply:
Ah, I see. They use `${PREFIX}` instead of `$(PREFIX)`. Adding this might work:
```ruby
ENV["PREFIX"] = prefix
```